config/checkstyle/checkstyle.xml

```diff
@@ -26,7 +26,9 @@
   <module name="FileTabCharacter">
     <property name="eachLine" value="true"/>
   </module>
-
+  <module name="BeforeExecutionExclusionFileFilter">
+    <property name="fileNamePattern"  value=".*[\\/]generated[\\/].*$||.*[\\/]controllers[\\/].*$"/>
```

Comment:
does this exclude files under "controllers" from being checking? is that something we want/need?

Reply:
yeah, the controllers have annotations etc in them that was generated from openapiGenerator, so it generates lines that are longer than the recommended 120 characters etc. i can't remember if that was the only rule broken, so either we remove the rule or exclude the file.

Reply:
We can just suppress the line length check for those files

```
  <module name="SuppressionSingleFilter">
    <property name="files" value=".*[\\/]controllers[\\/].*$" />
    <property name="checks" value="LineLength" />
  </module>
```

Reply:
done